layouts/_default/baseof.html

```diff
@@ -120,6 +120,7 @@
                         <li class="divider"></li>
                         <li class="dropdown-header">Project</li>
                         <li><a href="/about_jena/about.html">About Jena</a></li>
+                        <li><a href="/about_jena/citing.html">Citing</a></li>
```

Comment:
Added a new section, but this could also go in the About Jena section. However, I think some users would miss it (or not search hard enough) and ask in the mailing list anyway?

Reply:
(It isn't showing in the staged build for me - should PRs from other repos show up there?)

I doubt people that don't search hard enough will look in a dropdown. Or look at the site :-) Best we can hope for is Google search for "jena citation".

Adding to baseof.html adds it to every page.

Maybe one the "about" page is enough?